### DISCUSSION TITLE
Incorrect UI labels and empty pages

### DISCUSSION THREAD

**Discussion by kullackov** [subquery/subql#1062] [Frontier TestNet - Bug Reporting]
1. It is possible to stake 1000 SQT, but the label says that "Staking token should be large than 1000 SQT"
![image](https://user-images.githubusercontent.com/48903655/169800645-800614bb-4efd-4d66-a617-7178ef64e342.png)

2. All rewards were claimed, but label says "You have an unclaimed reward"
![image](https://user-images.githubusercontent.com/48903655/169800919-3c2290d4-01f9-4045-8f07-b5c719a77a7a.png)

3. There were 10 pages at frontier.subquery.network/staking/ndexing and the last of them were empty.
![image](https://user-images.githubusercontent.com/48903655/169801337-39aaf966-0915-4827-881d-2f773d1351a0.png)

**Comment by mzxyz** [comment]
Thanks for reporting this issue, we will fix these bugs  @HZHAIX @lilian79511 @CaiYiLiang
